locales/en.yml

```diff
@@ -22,7 +22,9 @@ en:
       submit_title: Submit search query
       submit_label: Search
     targeted_content:
-      close_button_aria: Collapse %{title}
+      descriptive_label_show: show this section
```

Comment:
Need to decide on the appropriate wording here, i.e. show/hide, close/open, or expand/collapse.

Reply:
I like show and hide, the GDS stuff mentioned show/hide being more easily understood as it actually describes what is happening, the others are a bit more figurative

Reply:
Cool, I went for that for the same reason. I guess my follow up question is if we're happy on sticking with "Close" for the button at the bottom of the UI. i.e. the visual label would read "Close" but the longer form aria text would say hide this section.